Library/Formula/llvm.rb

```diff
@@ -198,12 +196,6 @@ def install
 
     args << "-DLLVM_ENABLE_RTTI=On" if build.with? "rtti"
 
-    if build.with? "assertions"
-      args << "-DLLVM_ENABLE_ASSERTIONS=On"
-    else
-      args << "-DCMAKE_CXX_FLAGS_RELEASE='-DNDEBUG'"
```

Comment:
Oh, does that have to be added manually? I thought LLVM would build without assertions by default because this is building a Release version...